### PR TITLE
fixes issue #218

### LINKED ISSUE #218: Add meta tag and ogp tags inside the head section
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
We should add relevant meta tags and ogp tags inside the head section of the index.html template so that search engines can easily find out our website
**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by sarnava1**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Initially the page did not have required meta tags and ogp tags for a proper SEO. So in this PR I have added the required meta tags along with the ogps. This will make the ranking of our site better and during searches our site's ranking will be better.

#### Changes proposed in this pull request:

-Added meta tags
-Added ogp tags

Fixes #218

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -2,8 +2,25 @@
...
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <!-- meta tags for better SEO -->
+  <meta name="keywords" content="FOSSASIA, meilix-generator, Meilix-Generator, meilix, Meilix,ISO,
+  custom meilix, linux, personalised linux distribution,webapp,iso" />
+  <meta name="description" content="Meilix-Generator is a webapp which generates an iso with
+  a custom meilix. The main aim of the project is to help a user to make its personalised
+  linux distribution." />
+  <meta name="author" content="FOSSASIA" />
+  <!-- OGP -->
+  <meta property="og:locale" content="en_IN" />
```
Comment (tabesin) [inline templates/index.html]: Why en_in locale?

File: templates/index.html [hunk 2]
```diff
@@ -2,8 +2,25 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta content="IE=edge" http-equiv="X-UA-Compatible">
+  <meta charset="UTF-8" />
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <!-- meta tags for better SEO -->
+  <meta name="keywords" content="FOSSASIA, meilix-generator, Meilix-Generator, meilix, Meilix,ISO,
+  custom meilix, linux, personalised linux distribution,webapp,iso" />
```
Comment (tabesin) [inline templates/index.html]: you mention iso two times

Comment (tabesin) [reply templates/index.html]: Also Meilix Generator and meilix. Keywords should be case insensitive.

Comment (tabesin) [reply templates/index.html]: Comma setting is not uniform, with space or without but not both in the same line.

Comment (sarnava1) [reply templates/index.html]: @tabesin I rectifying the issues.

Comment (sarnava1) [reply templates/index.html]: I have made the comma settings uniform now.

Comment (sarnava1) [reply templates/index.html]: I deliberately mentioned "ISO" once and "iso" another time. Same is the case with meilix and Meilix Generator. I actually thought of all the keywords which a person can think off about meilix. So even if he gives the keyword of iso or ISO, meilix or Meilx search engines will show our site at the top. So, should I change them and make them case sensititve or keep them as they are now.

Comment (tabesin) [reply templates/index.html]: Okay, I approve the PR as it is and you could fix the cosmetic issue in the next PR.

**Pull request merged**